modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/modal/Modal.js

```diff
@@ -246,6 +255,7 @@ const Modal = ({
 													? 'button'
 													: button.type
 											}
+											{...button.otherProps}
```

Comment:
Necessary for setting `autoFocus` on the button and it allows more extensibility